sphinx-docs/usermanual/userman_learner.rst

```diff
@@ -0,0 +1,49 @@
+Learner User Manual
+=======================
+
+.. toctree::
+   :maxdepth: 1
+
+ 	userman_learner
+
+
+
+Introduction
```

Comment:
Consider breaking the various sections of this topic into subtopics (as separate files). There should be a directive for including another .rst file.